_includes/downloads-scala3.html

```diff
@@ -0,0 +1,136 @@
+Scala binaries for <strong>{{page.release_version}}</strong> are available at <a href="https://github.com/lampepfl/dotty/releases/tag/{{page.release_version}}">github</a>.
+
+<h2>Getting Started with Scala</h2>
+<div class="install-steps">
+<p>
+  Depending on your experience and your requirements, there exist two ways to
+  install Scala 3 on your computer,
+  either through <a href="https://www.scala-sbt.org/download.html">sbt</a>, or
+  through <a href="https://get-coursier.io/docs/cli-installation">Coursier</a>.
+</p>
+
+<div class="download-options">
+  <div class="download-left">
+    <span class="or">or</span>
+    <a href="https://get-coursier.io/docs/cli-installation" class="btn-download">
+      <i class="fa fa-download"></i>
+      <span>Download Coursier</span>
+    </a>
+    <div class="text-variant">
+      <p>Best if you want to get <strong>started quickly</strong>.</p>
+      <p><i>Coursier will install all the necessary dependencies for you, so after
+         installing Coursier you are ready to go.</i></p>
+    </div>
+  </div>
+  <div class="download-right">
+    <a href="https://www.scala-sbt.org/download.html"" class="btn-download">
```

Comment:
```suggestion
    <a href="https://www.scala-sbt.org/download.html" class="btn-download">
```

Reply:
Sorry -- I fixed it, but forgot to save before adding... My bad.